integration_tests/models/custom_calendar.sql

```diff
@@ -20,7 +20,11 @@ final as (
         {% endif %}
         cast({{ dbt_utils.date_trunc('month', 'date_day') }} as date) as date_month,
         cast({{ dbt_utils.date_trunc('quarter', 'date_day') }} as date) as date_quarter,
-        cast({{ dbt_utils.date_trunc('year', 'date_day') }} as date) as date_year
+        cast({{ dbt_utils.date_trunc('year', 'date_day') }} as date) as date_year,
+        case 
+            when dayname(to_date(date_day)) in ('Sat','Sun') then true
+            else false 
+        end as is_weekend
```

Comment:
Surprised that `is_weekend` isn't defined in some cross-database macro somewhere!

We should probably get that defined so this functionality works cross-database.

Reply:
It very well could be! To be honest I only added this field to test out functionality around bringing in dimensions from calendar tables.